guide/how-it-works/ecash/introduction.md

```diff
@@ -0,0 +1,242 @@
+
+Since no accounts or personal information are needed; the mint cannot see your balance, tokens, or payment details, offering a highly private and efficient method for digital payments.
+
+## How bitcoin-backed ecash works
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/how-ecash-works.png"
+   retina = "/assets/images/guide/how-it-works/ecash/how-ecash-works@2x.png"
+   mobile = "/assets/images/guide/how-it-works/ecash/how-ecash-works@2x.png"
+   mobileRetina = "/assets/images/guide/how-it-works/ecash/how-ecash-works@2x.png"
+   alt-text = "Diagram showing three mints (Mint 1, Mint 2, Mint 3) managing digital ecash payments with groups of users. Each mint is connected to a group of users represented by icons holding digital cash. The mints are interconnected with lightning bolts symbolizing fast or real-time payments, and the Bitcoin symbol appears beneath each mint, indicating that the system is based on Bitcoin technology."
+   caption = "Three mints manage ecash tokens for users, connected by the lightning network."
+   width = 800
+   height = 508
+   layout = "full-width"
+%}
+
+Bitcoin-backed ecash works by using protocols like [Fedimint]({{ '/guide/how-it-works/ecash/fedimint' | relative_url }}) and [Cashu]({{ '/guide/how-it-works/ecash/cashu/' | relative_url }}). Users deposit bitcoin, and in return, they receive bitcoin-backed ecash tokens. Users can then spend and receive these ecash tokens, which remain anonymous and unlinkable to their identity, enabling private payments within the bitcoin ecosystem.
+
+### Key benefits
+
+* **Faster payments** - Ecash systems process payments almost instantly since they rely on a centralized mint. They do not require [network-wide consensus]({{ '/guide/how-it-works/nodes' | relative_url }}) or routing through multiple [lightning nodes]({{ '/guide/how-it-works/nodes/#routing-nodes' | relative_url }}).
+
+* **Higher volume** - The centralized mint allows ecash systems to handle a larger volume of payments efficiently. The bottlenecks typical of decentralized networks like the lightning network are avoided.
+
+* **Simplified processing** - payments are validated and settled internally within the mint. This reduces complexity and avoids potential delays associated with routing and liquidity issues in the lightning network.
+
+* **Consistent speed** - Ecash systems offer consistent payment speeds regardless of network conditions. Unlike the lightning network, where payment speeds can be affected by channel capacity, node connectivity, and node hardware.
+
+* **Enhanced privacy** - [Blinded signatures](https://en.wikipedia.org/wiki/Blind_signature) safeguard user identities and total balances. While the mint can see amounts for individual payments (like paying an invoice or swapping proofs), it cannot associate these with specific users or determine anyone's total ecash holdings, as there are no user accounts.
+
+---
+### Ecash protocols
+
+{% include picture.html
```

Comment:
The image is definitely decorative. I think adding it here is nice because it gives the page a more personality and sense of design identity. In my opinion it doesn’t impact readability too much.  Like the comment above, if this is more about aesthetics, I’d rather keep it. But if you think it hurts readability enough to affect the guide, I’m okay with removing it.